### PR TITLE
pmx_biobb -- Update meta.yaml -- linux python 3.7

### PR DESCRIPTION
**Pull request by PauAndrio**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/pmx_biobb/meta.yaml [hunk 1]
```diff
@@ -24,13 +24,13 @@ requirements:
     - {{ compiler('fortran') }}
     - make
   host:
-    - python
+    - python =3.7
```
Comment (bgruening) [inline recipes/pmx_biobb/meta.yaml:27]: Why is this needed, its not recommended to pin a python version.

Comment (PauAndrio) [reply recipes/pmx_biobb/meta.yaml:27]: Hi,

This is the only workaround I found to create a package compatible with Python versions from 3.7 to 3.11. In this case

----Dependency----------------Python ver-------------Anaconda package created

- python [no pinned vers] ------3.8-----> [linux-64/pmx_biobb-4.1.2-py38h0928705_2.tar.bz2](https://anaconda.org/bioconda/pmx_biobb/4.1.2/download/linux-64/pmx_biobb-4.1.2-py38h0928705_2.tar.bz2)
- python [no pinned vers] ------3.9-----> [linux-64/pmx_biobb-4.1.2-py39hef94ad9_2.tar.bz2](https://anaconda.org/bioconda/pmx_biobb/4.1.2/download/linux-64/pmx_biobb-4.1.2-py39hef94ad9_2.tar.bz2)
- python [no pinned vers] ------3.10----> [linux-64/pmx_biobb-4.1.2-py310h95c5ba8_2.tar.bz2](https://anaconda.org/bioconda/pmx_biobb/4.1.2/download/linux-64/pmx_biobb-4.1.2-py310h95c5ba8_2.tar.bz2)

- python =3.7 ------------------3.7-----> [linux-64/pmx_biobb-4.1.2-py37h8537716_1.tar.bz2](https://anaconda.org/bioconda/pmx_biobb/4.1.2/download/linux-64/pmx_biobb-4.1.2-py37h8537716_1.tar.bz2)
- python >=3.7 -----------------3.11-----> [linux-64/pmx_biobb-4.1.2-py311h8537716_0.tar.bz2](https://anaconda.org/bioconda/pmx_biobb/4.1.2/download/linux-64/pmx_biobb-4.1.2-py311h8537716_0.tar.bz2)

Is there any other "official/recommended" way to force bioconda to create the 5 packages, not using pining trick?
